x-pack/plugins/profiling/public/embeddables/profiling_embeddable_search_bar.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProfilingSearchBarFilter } from '@kbn/observability-shared-plugin/public';
+import React from 'react';
+import { v4 } from 'uuid';
+import { ProfilingSearchBar } from '../components/profiling_app_page_template/profiling_search_bar';
+
+interface Props {
+  onQuerySubmit: (filters: ProfilingSearchBarFilter) => void;
+  kuery?: string;
+}
+
+export function ProfilingEmbeddableSearchBar({ onQuerySubmit, kuery = '' }: Props) {
+  return (
+    <ProfilingSearchBar
+      showDatePicker={false}
```

Comment:
why do you keep the APM date picker when you can show the date picker in the profiling search bar similar to profiling page? 

![Screenshot 2023-11-20 at 14 40 16](https://github.com/elastic/kibana/assets/3369346/3fa55215-35a8-4429-8ede-358465b59b8f)

I think as embeddable it should handle it.

Reply:
I haven't thought about this actually. I'll see how that's going to look like and come back to you.

Reply:
Just a note to this, in case we decide to have the date picker as part of the embeddable, it should have a param to pass the initial date values to keep it in sync with other tabs that use their own date pickers.

Reply:
It will be @mykolaharmash , you can pass in the time range to the component.